src/Sarif/Autogenerated/IRule.cs

```diff
@@ -40,6 +40,11 @@ public partial interface IRule
         IDictionary<string, string> MessageFormats { get; }
 
         /// <summary>
+        /// A value specifying the configuration state of the rule.
```

Comment:
>configuration state of the rule [](start = 35, length = 31)

I understand why the member name is called `Configuration`. But I think the comment should be more explicit "A value specifying whether the rule is enabled."